ReleaseTooling/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -543,6 +546,40 @@ struct FrameworkBuilder {
               "\(error.localizedDescription)")
           }
         }
+        do {
+          // Frameworks built from only Swift sources will contain only two
+          // headers: the CocoaPods-generated umbrella header and the
+          // Swift-generated Swift header. If the framework's `Headers`
+          // directory contains more than two resources, then it is assumed
```

Comment:
What happens with an Objective-C only framework with one or two headers?

Reply:
Good question. So this code shouldn't run on an Objective-C only framework. Line [487](https://github.com/firebase/firebase-ios-sdk/pull/11919/files#diff-f4eb4ff5ec89af999cbe8fa3ffe5647d7853ffbc9c1515b337ca043c684b6bb4R487) will early exit if there is no Swift module.

Reply:
I added some more details in fe1e5d9.